query/src/lib/signals.ts

```diff
@@ -72,6 +72,7 @@ export function intersectResults<
       isFetching: toArray.some((v) => v().isFetching),
       error: toArray.find((v) => v().isError)?.error,
       data: undefined,
+      refetch: () => Promise.all(toArray.map(v => v().refetch())),
```

Comment:
The only concern I have with this is that we create a new method each time it runs even though the keys are initialized once.

Reply:
Is creating a method that expensive? Do you have another idea how to achieve the refetch behavior?

Reply:
You can create it once in the closure (above the computed) and return the same reference each time

Reply:
Please also add an example to our playground